### DISCUSSION TITLE
problem with update 1.46

### DISCUSSION THREAD

**Discussion by EdwardKiefer** [Rem0o/FanControl.Releases#1534] [General]
auto updater updates to 1.46 from 1.45 but on restart 1.45 shows and wants to update to 1.46 again (update loop).

**Comment by Byleth-Eisner** [comment]
I am also having the same issue (update loop).

Edit: the problem has now been resolved for me.

**Reply by EdwardKiefer** [reply]
Yes, it is fixed now.
